mkdocs.yml

```diff
@@ -1,8 +1,13 @@
 site_name: Freqtrade
 nav:
     - Home: index.md
-    - Installation Docker: docker.md
-    - Installation: installation.md
+    - Installation Docker: 
+        - Quickstart: docker.md
```

Comment:
Should this quickstart be outside of the docker installation subsection?

otherwise ALL installation pages are in subsections - making it a 2-click navigation to get to any installation page.

Reply:
I agree. Joined Quickstart with docker and moved out of installation.